modules/compute/vm-instance/variables.tf

```diff
@@ -245,6 +245,12 @@ variable "guest_accelerator" {
   default = null
 }
 
+variable "automatic_restart" {
+  description = "Specifies if the instance should be restarted if it was terminated by Compute Engine (not a user). Defaults to true."
```

Comment:
I think default to true may be misleading. More like, by default we will turn off when appropriate and otherwise use the platform default.

Reply:
> default we will turn off when appropriate and otherwise use the platform default.

I'm not sure if it adds any clarity, I would just remove the mention of "default" all together.